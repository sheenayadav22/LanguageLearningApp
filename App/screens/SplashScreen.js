import * as React from 'react';
import { View, Text } from 'react-native';
import main from '../styles/main';

function SplashScreen() {
  return (
    <View style={main.centered} >
      <Text>Loading...</Text>
    </View>
  );
}

export default SplashScreen;
