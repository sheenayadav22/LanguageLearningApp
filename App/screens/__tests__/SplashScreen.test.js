import React from 'react';
import renderer from 'react-test-renderer';
import SplashScreen from '../SplashScreen';

it('renders correctly', () => {
  const tree = renderer.create(<SplashScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
