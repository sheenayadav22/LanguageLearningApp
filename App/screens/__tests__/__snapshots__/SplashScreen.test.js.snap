// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 0.9,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Text>
    Loading...
  </Text>
</View>
`;
